### PR TITLE
Feature/ti2023 10/create register page frontend

### PR DESCRIPTION
**Pull request by Filip-B-32**

### CODE REVIEW

File: event-planner-frontend/src/App.tsx [hunk 1]
```diff
@@ -1,32 +1,50 @@
...
-      .catch((err) => { console.log("not ok", err.response.data); })
-
-  }, [])
-
-
-  return <div>Event Planner</div>
+    userService
+      .getUsers()
+      .then((res) => {
+        console.log(res.data);
```
Comment (HellHound97) [inline event-planner-frontend/src/App.tsx]: remove console.log

File: event-planner-frontend/src/App.tsx [hunk 2]
```diff
@@ -1,32 +1,50 @@
...
+      })
+      .catch((err) => {
+        console.log(err.response.data);
```
Comment (HellHound97) [inline event-planner-frontend/src/App.tsx]: remove console.log

File: event-planner-frontend/src/App.tsx [hunk 3]
```diff
@@ -1,32 +1,50 @@
...
+      });
+
+    let data: object = { name: 'Mirel' };
+    userService
+      .createUser(data)
+      .then((res) => {
+        console.log(res.data);
```
Comment (HellHound97) [inline event-planner-frontend/src/App.tsx]: remove console.log

File: event-planner-frontend/src/App.tsx [hunk 4]
```diff
@@ -1,32 +1,50 @@
...
+      })
+      .catch((err) => {
+        console.log(err.response.data);
```
Comment (HellHound97) [inline event-planner-frontend/src/App.tsx]: remove console.log

File: event-planner-frontend/src/App.tsx [hunk 5]
```diff
@@ -1,32 +1,50 @@
...
+      });
+
+    userService
+      .updateUser(10, data)
+      .then((res) => {
+        console.log(res.data);
```
Comment (HellHound97) [inline event-planner-frontend/src/App.tsx]: remove console.log

File: event-planner-frontend/src/App.tsx [hunk 6]
```diff
@@ -1,32 +1,50 @@
...
+      })
+      .catch((err) => {
+        console.log(err.response.data);
```
Comment (HellHound97) [inline event-planner-frontend/src/App.tsx]: remove console.log

**Comment by HellHound97** [comment]
remove console.log from frontend

**Review by HellHound97 — APPROVED** [review]

**Review by alexandratodea — APPROVED** [review]

**Pull request merged**